frontend/src/components/dialogs/SecretDialogOpenstack.vue

```diff
@@ -138,16 +162,19 @@ SPDX-License-Identifier: Apache-2.0
 <script>
 import { mapGetters } from 'vuex'
 import SecretDialog from '@/components/dialogs/SecretDialog'
-import { required, requiredIf, requiredUnless, and } from 'vuelidate/lib/validators'
-import { nilIf } from '@/utils/validators'
+import { required, requiredIf } from 'vuelidate/lib/validators'
 import { getValidationErrors, setDelayedInputFocus } from '@/utils'
 import HintColorizer from '@/components/HintColorizer'
 import ExternalLink from '@/components/ExternalLink'
 
 const requiredMessage = 'You can\'t leave this empty.'
-const requiredUserPassMessage = 'You can\'t leave this empty unless application credentials are used.'
-const requiredAppCredMessage = 'Required for application credentials.'
-const authOrAppCredentialsMessage = 'Application credentials are used only if no authentication credentials are given.'
+const requiredUserPassMessage = 'Required for technical user authentication'
+const requiredAppCredMessage = 'Required for application credential authentication'
```

Comment:
Is it application credential or application credentials? We should use it consistently. And I would not abbreviate with `cred`.

Reply:
well the `Application Credentials` consist of three Credentials: `applicationCredentialID`, `applicationCredentialName`, `applicationCredentialSecret` so when we talk about them as a whole we call them `Application Credentials` while one of them is an `Application Credential`, right?